### DISCUSSION TITLE
Can I use a 64-bit pi?

### DISCUSSION THREAD

**Discussion by bonddenhalter** [theyosh/TerrariumPI#731] [General]
I am considering creating a setup with a Raspberry Pi 4 Model B, which is a 64-bit system. Comparing what I've read on the README as well as https://theyosh.github.io/TerrariumPI/install/, I need to have a Raspberry Pi 4, running OS Lite, but 64-bit is not supported. Is that accurate?

**Comment by theyosh** [comment]
Hi,

64bit is not tested. And as far I know, there are issues with the RPI camera. So I did not test it. Also, I do not know if all hardware will work. And if all python modules do compile for 64bit.

So, for now, you could give it a try if you are a bit handy with coding and debugging. Else I would suggest to stick to the old legacy image, because that is 100% tested and supported.
